src/universal-app/app/splitter/split-handle.directive.ts

```diff
@@ -0,0 +1,32 @@
+import {Directive, ElementRef, Output} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+
+import 'rxjs/add/observable/fromEvent';
+import 'rxjs/add/operator/takeUntil';
+import 'rxjs/add/operator/switchMap';
+import 'rxjs/add/operator/map';
+
+@Directive({
+  selector: '[ngxSplitHandle]',
+  host: {
+    class: 'ngx-split-handle',
+    title: 'Drag to resize'
+  }
+})
+export class SplitHandleDirective {
+
+  @Output() drag: Observable<{ x: number, y: number }>;
+
+  constructor(ref: ElementRef) {
+    const getMouseEventPosition = (event: MouseEvent) => ({x: event.movementX, y: event.movementY});
+
+    const mouseup$ = Observable.fromEvent(document, 'mouseup');
```

Comment:
@ThomasBurleson I am not sure if platform-server has `ownerDocument` property implementation.